glib/src/value.rs

```diff
@@ -930,6 +936,33 @@ impl SetValueOptional for str {
     }
 }
 
+impl<'a> FromValue<'a> for Value {
+    unsafe fn from_value(value: &'a Value) -> Self {
+        from_glib_none(
+            gobject_ffi::g_value_get_boxed(value.to_glib_none().0) as *mut gobject_ffi::GValue
```

Comment:
Not sure why I wrote `*mut`, this probably has to be `*const`?

Reply:
See below. `const` is more correct but it also doesn't really make a difference